### ISSUE TITLE
Bitcoin transaction neither confirmed yet for over 16 hrs nor found on InSight

### ISSUE THREAD

**Issue by luowei0458** [bitpay/copay#7587]
I sent 1 BTC to the same destination by 2 transactions yesterday. One was 0.099 BTC with 0.001662 BTC transaction fee on 5:24 PM GMT, the other was 0.8977 BTC with 0.001524 BTC transaction fee on 6:38 PM GMT. The previous transaction completed successfully in 1 hour after sending, somehow, the later one neither has been confirmed yet for over 16 hours nor was found its record on InSight website.

Any advice or help would be gratefully appreciated. Thanks!

**Comment by xereaz** [comment]
had the same problem but copay was holding an additional 1.6 btc of mine I just put my seed words into my cellium and the transactions all came up and was able to atleast recover the 1.6 however the .4 is still unconfirmed

**Comment by luowei0458** [comment]
Got my transaction records on https://blockchain.info/address/1CeS8eWZmvJ6gScsNjWgtxG184kXDA9Rxr. Someone told me that my asset was very likely to be stuck in the network due to the relatively low transaction fee I paid. So it seems that I have nothing to do but wait for the confirmations.

**Comment by luowei0458** [comment]
## # I dont know whether I've got my bitcoins back as I saw this in my Copay wallet.
![cp1](https://user-images.githubusercontent.com/34290749/34283649-ecb9ed2a-e707-11e7-8534-44c8f2b29a80.jpg)
## When I touch Personal Wallet (the box below the funds amount) to see the details as below, it looks normal as well. 
![cp2](https://user-images.githubusercontent.com/34290749/34283745-89c11a3a-e708-11e7-96b4-6213cd7c9fb8.jpg)
## But when I touch the REFRESH icon on the upper right corner of the page for several times, strange and terrible things happened.
![cp3](https://user-images.githubusercontent.com/34290749/34283891-57547046-e709-11e7-8eab-784878267d19.jpg)
## It's lucky (or more strange) that if I refresh more, the funds will be back again.
![cp4](https://user-images.githubusercontent.com/34290749/34284063-3f83100c-e70a-11e7-8a7c-4696bddf37d4.jpg)
# Is there anyone who met this kinda things as well? Is there anyone who can tell me why? I'll be appreciate to any help for me. Thank you.

**Comment by notf0und** [comment]
Import your mnemonic seed into Samourai Wallet and forgive this project, they are more interested on btrash right now.

**Comment by klopferrr** [comment]
I have the same Problem as you luowei0458

**Comment by luowei0458** [comment]
I just got my 2nd transaction confirmed finally. It took almost 54 hours to reach the destination. And the transferring history shown on the Copay Wallet seems OK as usual. So just give a proper or higher fee for bitcoin transaction if you are in a hurry. By viewing the closed issues and the Copay FAQ in this forum, I got this comment: https://github.com/bitpay/copay/issues/5164#issuecomment-265569494. Hope to help you all.
